pennylane/tape/measure.py

```diff
@@ -169,7 +169,12 @@ def expand(self):
     def queue(self):
         """Append the measurement process to an annotated queue."""
         if self.obs is not None:
-            qml.tape.QueuingContext.update_info(self.obs, owner=self)
+            try:
+                qml.tape.QueuingContext.update_info(self.obs, owner=self)
+            except ValueError:
+                self.obs.queue()
+                qml.tape.QueuingContext.update_info(self.obs, owner=self)
```

Comment:
An edge case discovered when attempting to get the `qml.map()` tests to pass. The new queuing refactor did not take into account observables created _outside_ of a queuing context!

This allows previously instantiated observables to be used within a QNode.

Reply:
I assume the `update_info` fails above because `obs` is not in queue?

Reply:
Yep! Exactly. It might be a flaw in the queuing refactor design; the queuing refactor explicitly assumes that all nested objects exist in the queue prior to other objects annotating the queue.